tests/checkup_test.go

```diff
@@ -105,8 +105,6 @@ var _ = Describe("Execute the checkup Job", func() {
 		Expect(configMap.Data).NotTo(BeNil())
 		Expect(configMap.Data["status.succeeded"]).To(Equal("true"), fmt.Sprintf("should succeed %+v", prettifyData(configMap.Data)))
 		Expect(configMap.Data["status.failureReason"]).To(BeEmpty(), fmt.Sprintf("should be empty %+v", prettifyData(configMap.Data)))
-		Expect(configMap.Data["status.result.trafficGeneratorNode"]).ToNot(BeEmpty(),
```

Comment:
why is this removed?

Reply:
Because there is no traffic generator pod, so it is not scheduled to any node.